### PR TITLE
Feature/rad 999: added scroll animation to certification section

### PR DESCRIPTION
**Pull request by rogdevil**

### CODE REVIEW

File: src/components/homepage/homepage6.scss [hunk 1]
```diff
@@ -1,3 +1,64 @@
+@import "../../styles/mixins";
+@import "~sass-rem";
+
 .scrolling-images-container {
   display: block;
+  margin: 120px auto;
```
Comment (prakhar-ira) [inline src/components/homepage/homepage6.scss:6]: Use rem

File: src/components/homepage/homepage6.scss [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+  .homepage6_title {
+    background: linear-gradient(92.15deg, #b62bf5 0%, #e62e2c 100%);
+    @include backgroundClip;
+    font-size: 80px;
```
Comment (prakhar-ira) [inline src/components/homepage/homepage6.scss:10]: Rem

File: src/components/homepage/homepage6.tsx [hunk 3]
```diff
@@ -1,19 +1,66 @@
...
+  });
+
+  const podiumDataRef = useIntersectionObserver(podiumTriggerRef, {
+    threshold: 1,
+    freezeOnceVisible: false,
+    root: document.getElementById("home_scroller")
+  });
+
+  useEffect(() => {
+    console.log(dataRef?.intersectionRatio);
```
Comment (prakhar-ira) [inline src/components/homepage/homepage6.tsx:29]: Remove console

File: src/components/homepage/homepage6.scss (hunk 1 above)
Comment (rogdevil) [reply src/components/homepage/homepage6.scss:6]: done

File: src/components/homepage/homepage6.scss (hunk 2 above)
Comment (rogdevil) [reply src/components/homepage/homepage6.scss:10]: done

File: src/components/homepage/homepage6.tsx (hunk 3 above)
Comment (rogdevil) [reply src/components/homepage/homepage6.tsx]: done

**Pull request merged**